interview-questions.md

```diff
@@ -15,54 +15,52 @@ INSTRUCTOR EXAMPLE: What is a conditional statement?
 
 1. What is git? What is the difference between git and Github?
 
-  Your answer:
+  Your answer: GitHub is the online platform that is used to collaborate and pass code to peers and the public. Git is the process and language that is used to communicate to GitHub. 
 
-  Researched answer:
+  Researched answer: Git is the version control system that allows the user to access the previous versions and history of a project using the command line on their local computer.  GitHub is the online collaboration platform used to refine and share the project. 
 
 
 
 2. Which JavaScript operators will return a Boolean value?
 
-  Your answer:
-
-  Researched answer:
+  Your answer: Relative Operators and Logical Operators will return a boolean value.
```

Comment:
Make sure to expand on this.  In an interview, you will want to provide as much context as possible to show your understanding, otherwise it can come off as uninterested.